packages/wonder-blocks-dropdown/src/components/single-select.js

```diff
@@ -157,13 +179,6 @@ type State = {|
     openerElement: ?HTMLElement,
 |};
 
-type DefaultProps = {|
```

Comment:
**note:** Found an issue with Storybook + [Indexed Access Types](https://flow.org/en/docs/types/indexed-access/), so I had to move the default props above.